### DISCUSSION TITLE
Support for higher resolution LCDs?

### DISCUSSION THREAD

**Discussion by mycowx** [alf45tar/PedalinoMini#429] [Q&A]
Gathering parts for a PedalinoMini build and ran across a LilyGo LCD board with a 170*320 Resolution 1.9 inch ST7789 LCD Display.  Would this display be supported with the current firmware?  

Here's a link to the board I am looking at:
https://www.amazon.com/LILYGO-T-Display-Arduino-Development-CH9102F/dp/B09J112YR7?th=1

Thanks!

**Comment by alf45tar** [comment]
No plan for the moment.
